compiler/rustc_metadata/src/native_libs.rs

```diff
@@ -190,12 +209,22 @@ impl<'tcx> ItemLikeVisitor<'tcx> for Collector<'tcx> {
                             }
                         }
                     }
+                    if has_duplicate_modifiers {
+                        let msg =
+                            "same modifier is used multiple times in a single `modifiers` argument";
+                        sess.span_err(item.span(), msg);
+                    }
                 } else {
                     let msg = "must be of the form `#[link(modifiers = \"...\")]`";
                     sess.span_err(item.span(), msg);
                 }
             }
 
+            if modifiers_count > 1 {
+                let msg = "multiple `modifiers` arguments in a single `#[link]` attribute";
+                sess.span_err(m.span, msg);
+            }
```

Comment:
I think this should only be an error on stable right? On nightly, we currently accept `#[link(name = "foo", kind = "static", modifiers = "-bundle,+whole-archive")]` and I don't think we need to change that, do we?

Reply:
No, this is about the `#[link(modifiers = "...", modifiers = "...", ...)]` case (there's a test for it).

Reply:
Ah, got it. Thanks!